### ISSUE TITLE
Display buttons above the timecourse plot to "scroll" between gene inputs

### ISSUE THREAD

**Issue by bhavyaac** [fungenomics/braindex#38]
I plan use to simple action buttons (i.e. same kind of button as the update button) with labels such as "<- Previous gene" and "Next gene ->" to iterate through the list of input genes and plot each one. It will probably be easiest to implement this in a way where pressing "Previous" on the first gene loops around to the last gene in the list, and "Next" on the last one goes to the first. (I may change this later if it's confusing for the user.) 

I may want to change my gene validation step so that errors will only show up when trying to plot a gene that is not found in the data set, but the other plots will still be shown.

Because these buttons will update the plot immediately, I'm planning to place the buttons in the circled area of the screenshot, just above the plot, and close to the interactivity toggle (which is also immediate response):
![image](https://user-images.githubusercontent.com/26724860/125137225-44ff3380-e0da-11eb-88a4-8a68c399e000.png)

**Comment by bhavyaac** [comment]
I was able to set up something similar, but using a dropdown selection instead of action buttons (action buttons are really hard to use in this case because they don't store meaningful values). The selection choices match the genes provided as input, and they change dynamically with the update button. Here is a gif preview, let me know if you have any feedback @sjessa! (I've tested it and it also works with the interactive plot as well as with an uploaded file.)
![select_timecourse_gene](https://user-images.githubusercontent.com/26724860/125141643-9eb82b80-e0e3-11eb-9a46-ef88fcb86553.gif)
